server/routes/search/handlers/probationSearch.ts

```diff
@@ -1,42 +1,54 @@
 import { Request, Response } from 'express'
 import SearchService from '../../../services/searchService'
 import statusConfig from '../../../licences/licenceStatus'
+import { ProbationSearchResult } from '../../../@types/licenceApiClientTypes'
```

Comment:
when you're just importing things for their types, you can `import type` or if you have a mixture of types or non types: `import { type bob, jim, type sally}`.

This means that the transpiled code doesn't actually import the module unless it needs to, and it won't trigger any side effects

Reply:
Done